[BUG] Scorer utils is not exported from the evals package

### Describe the Bug

When following https://mastra.ai/en/examples/scorers/tool-call-accuracy and importing the scorer utils 
`import { createAgentTestRun, createUIMessage, createToolInvocation } from "@mastra/evals/utils";` I get a ts error `Cannot find module '@mastra/evals/utils' or its corresponding type declarations.ts(2307)`

Trying to run my spec file with the example tool call accuracy code based test results in `Error: Missing "./utils" specifier in "@mastra/evals" package`


### Steps To Reproduce

1. Install @mastra/core@latest @mastra/evals@latest
2. Follow https://mastra.ai/en/examples/scorers/tool-call-accuracy#code-based-scorer-examples
3. Run the scorer and get error `Error: Missing "./utils" specifier in "@mastra/evals" package`

### Expected Behavior

Utils are exported form the package 

### Environment Information

- @mastra/core@0.16.2
- @mastra/evals@0.13.5
- node@20.19.0

Not running agent under Mastra, just using Vitest to assert against code based scorer as a trial

### Verification

- [x] I have searched the existing issues to make sure this is not a duplicate
- [x] I have included sufficient information for the team to reproduce and understand the issue

--- a/.changeset/solid-chefs-trade.md
+++ b/.changeset/solid-chefs-trade.md
@@ -1,0 +1,5 @@
+---
+'@mastra/evals': patch
+---
+
+Export scorer utils from @mastra/evals/scorers/utils

--- a/docs/src/content/en/examples/scorers/tool-call-accuracy.mdx
+++ b/docs/src/content/en/examples/scorers/tool-call-accuracy.mdx
@@ -27,7 +27,7 @@ The code-based scorer provides deterministic, binary scoring (0 or 1) based on e
 
 ```typescript
 import { createToolCallAccuracyScorerCode } from "@mastra/evals/scorers/code";
-import { createAgentTestRun, createUIMessage, createToolInvocation } from "@mastra/evals/utils";
+import { createAgentTestRun, createUIMessage, createToolInvocation } from "@mastra/evals/scorers/utils";
 ```
 
 ### Correct tool selection
@@ -482,8 +482,3 @@ const customModelScorer = createLLMScorer({
 - User intent alignment
 - Context-aware evaluation
 - Handling edge cases
-
-<GithubLink
-  marginTop='mt-16'
-  link="https://github.com/mastra-ai/mastra/blob/main/examples/basics/scorers/tool-call-accuracy"
-/>

--- a/packages/evals/package.json
+++ b/packages/evals/package.json
@@ -79,6 +79,16 @@
         "default": "./dist/scorers/code/index.cjs"
       }
     },
+    "./utils": {
+      "import": {
+        "types": "./dist/scorers/utils.d.ts",
+        "default": "./dist/scorers/utils.js"
+      },
+      "require": {
+        "types": "./dist/scorers/utils.d.ts",
+        "default": "./dist/scorers/utils.cjs"
+      }
+    },
     "./package.json": "./package.json"
   },
   "scripts": {

--- a/packages/evals/package.json
+++ b/packages/evals/package.json
@@ -79,7 +79,7 @@
         "default": "./dist/scorers/code/index.cjs"
       }
     },
-    "./utils": {
+    "./scorers/utils": {
       "import": {
         "types": "./dist/scorers/utils.d.ts",
         "default": "./dist/scorers/utils.js"

--- a/packages/evals/src/scorers/utils.ts
+++ b/packages/evals/src/scorers/utils.ts
@@ -23,12 +23,17 @@ export type TestCaseWithContext = TestCase & {
   context: string[];
 };
 
-export const createTestRun = (input: string, output: string, context?: string[]): ScoringInput => {
+export const createTestRun = (
+  input: string,
+  output: string,
+  additionalContext?: Record<string, any>,
+  runtimeContext?: Record<string, any>,
+): ScoringInput => {
   return {
     input: [{ role: 'user', content: input }],
     output: { role: 'assistant', text: output },
-    additionalContext: { context },
-    runtimeContext: {},
+    additionalContext: additionalContext ?? {},
+    runtimeContext: runtimeContext ?? {},
   };
 };
 

--- a/packages/evals/tsup.config.ts
+++ b/packages/evals/tsup.config.ts
@@ -9,6 +9,7 @@ export default defineConfig({
     'src/metrics/nlp/index.ts',
     'src/scorers/llm/index.ts',
     'src/scorers/code/index.ts',
+    'src/scorers/utils.ts',
   ],
   format: ['esm', 'cjs'],
   clean: true,

